src/Command.php

```diff
@@ -1,556 +1,741 @@
 <?php
 
 declare(strict_types=1);
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
 
-namespace Yiisoft\Db\ElasticSearch;
+namespace Yiisoft\Elasticsearch;
```

Comment:
Should we rename the package?

Reply:
If package not depend from `yiisoft/db`, then need remove `db-` prefix.